suuchi-core/src/test/scala/in/ashwanthkumar/suuchi/utils/ByteArrayUtilsSpec.scala

```diff
@@ -0,0 +1,15 @@
+package in.ashwanthkumar.suuchi.utils
+
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers._
+
+class ByteArrayUtilsSpec extends FlatSpec {
+
+  "ByteArrayUtils" should "say whether a given byte array starts with a specified byte array prefix" in {
```

Comment:
@ashwanthkumar I guess the test description wasn't self explanatory. I ll put that into a new test n push..